.github/workflows/build.yml

```diff
@@ -48,6 +48,31 @@ jobs:
           # This is the latest commit on the actual PR branch
           echo "IMAGE_TAG=${{ github.event.pull_request.head.sha }}" >> $GITHUB_ENV
 
+      - name: Set Environment variable
+        run: |
+          # tag build to the review env for vars and secrets
+          . terraform/workspace_variables/review.sh
+          echo "TF_VAR_key_vault_name=$TF_VAR_key_vault_name" >> $GITHUB_ENV
+          echo "TF_VAR_key_vault_infra_secret_name=$TF_VAR_key_vault_infra_secret_name" >> $GITHUB_ENV
+
+      - uses: azure/login@v1
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS_REVIEW }}
+
+      - name: Validate Azure Key Vault secrets
```

Comment:
Do we need to run validation here? We normally use it before feeding the secrets to terraform, and it's already done in the deploy workflow: https://github.com/DFE-Digital/find-teacher-training/blob/main/.github/workflows/deploy.yml#L105